daphne/src/roles.rs

```diff
@@ -198,9 +213,19 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
     /// Handle HTTP POST to `/upload`. The input is the encoded report sent in the body of the HTTP
     /// request.
     async fn http_post_upload(&self, req: &DapRequest<S>) -> Result<(), DapAbort> {
+        // Check whether the DAP version in the request is not Unknown.
```

Comment:
Here and below.
```suggestion
        // Check whether the DAP version indicated by the sender is supported.
```

Reply:
Thanks for pointing out the grammar issue :)